server/src/js/components/charts.js

```diff
@@ -310,6 +316,13 @@ function LineChart (containerId, data) {
   if (hasObserved) {
     clipInner.append('g').classed('observed observed-focus', true)
     const observedLine = lines.filter(l => l.type === 'observed')
+    if (!window.flood.model.station.isCoastal) {
+      for (let o = 0; o < observedLine.length; o++) {
```

Comment:
We are moving away from imperative coding towards a more declarative coding style

Please use `.map()` rather than a loop unless there is a reason to use the latter.